core/contracts/common/implementation/MultiRole.sol

```diff
@@ -10,17 +10,17 @@ library Exclusive {
         return roleMembership.member == memberToCheck;
     }
 
-    function resetMember(RoleMembership storage roleMembership, address newMember) internal {
+    function resetExclusiveMember(RoleMembership storage roleMembership, address newMember) internal {
```

Comment:
This is called in `scripts/TransferPermissions`

Reply:
I don't think it is. the `resetMember` function that is called in that script is from the `abstract contract MultiRole` on line [122](https://github.com/UMAprotocol/protocol/blob/5e33a7efa6955d4ee17e481822aa03dea9c7541e/core/contracts/common/implementation/MultiRole.sol#L122) and this function is not changed in this PR. Rather, this PR changes the function that `resetMember` calls within the context of the exclusiveRoleMembership.

Reply:
There is a unit test to ensure that `TransferPermissions` works correctly in  `test/scripts/TransferPermissions.js` which passed after this change.